File: src/backend/MyAIRunningMate/MyAIRunningMate.Contracts/Ingestion/Responses/IngestionViewResponse.cs
namespace MyAIRunningMate.Contracts.Ingestion.Responses;

public record IngestionViewResponse(
    string GarminActivityId,
    DateTime StartTime,
    string ExerciseType,
    double DistanceMetres,
    double DurationSeconds,
    double RecoveryTime,
    int NumberOfLaps,
    string ActivityStatus
);

File: src/frontend/MyAIRunningMate-UI/src/types/ingestion/ingestionViewResponse.ts
export interface IngestionViewResponse {
  garmin_activity_id: string;
  start_time: string;
  exercise_type: string;
  distance_metres: number;
  duration_seconds: number;
  recovery_time: number;
  number_of_laps: number;
  activity_status: string;
}